tox.ini

```diff
@@ -14,7 +14,14 @@ commands =
 deps =
     flake8==3.7.7
 
+[testenv:black]
+commands =
+    black . --check
```

Comment:
I think this should be `black --check .`. Note it might be safer to do:

```
 black --check pydruid setup.py tests
```

Reply:
Good! It was changed.